package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nhsuk-frontend",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
Does this need to be a minor version increase? It's a bugfix rather than new feature so I'd make it a patch update.